CHANGES.md

```diff
@@ -15,7 +15,34 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- Add mechanism to set CPU and memory limits on Jupyterlab containers based on Magpie group
+
+  Creates a new variable `JUPYTERHUB_GROUP_LIMITS` which sets resource limits for JupyterLab containers per 
+  Magpie group.
```

Comment:
I recommend renaming it to `JUPYTERHUB_GROUP_RESOURCE_LIMITS` to make it more explicit.